tests/modules/suite2p_registration/test_suite2p_registration_with_mocks.py

```diff
@@ -175,3 +176,32 @@ def test_suite2p_frame_rate_consistency(tmp_path, mock_ops_data):
                 MockSuite2PWrapper):
             runner = s2preg.Suite2PRegistration(input_data=args, args=[])
             assert runner.args['suite2p_args']['movie_frame_rate_hz'] == 11.0
+
+
+def test_suite2p_trim_frames_raises(tmp_path, mock_ops_data):
+    h5path = tmp_path / "mc_video.h5"
+    with h5py.File(str(h5path), "w") as f:
+        f.create_dataset("data", data=np.zeros((1000, 32, 32)))
+
+    outj_path = tmp_path / "output.json"
+    args = {"suite2p_args": {
+                "h5py": str(h5path),
+            },
+            'auto_remove_empty_frames': True,
+            'trim_frames_start': 10,
```

Comment:
Because I'm paranoid, can you repeat this test with `trim_frames_end >0`. Because I'm paranoid